dls-controls/malcolmjs
# Feature/style updates #304

## Description

Various style tweaks, including:
- Making alarm column in table widget 32px wide
- making alarm buttons 24px wide
- made table rows 36px high
- tweaked vertical size/padding on textinput and combobox
- removed background colour on LED div so highlight applies properly when selected
- autolayout button now raised + primary colour
- buttons for tables/layout etc now show "view" or "edit" based on the attribute's writeable flag.

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- navigate to localhost:3000/PANDA/layout/SEQ1/table
- study table for examples of styling

## Agile board tracking

connect to #304 
closes #304
